sympy/functions/elementary/tests/test_exponential.py

```diff
@@ -163,7 +163,7 @@ def test_log_values():
 
     assert exp(-log(3))**(-1) == 3
 
-    assert log(S.Half) == -log(2)
+    assert expand_log(log(S.Half)) == -log(2)
```

Comment:
BTW, this function `test_log_value` contains tests for values of log, may be move this test to https://github.com/sympy/sympy/blob/master/sympy/functions/elementary/tests/test_exponential.py#L331.

Or if want to test it, may be change it to `log(S.Half) == log(S.Half, evaluate=False)`, instead of using `expand_log`?

Reply:
Yes, comparing with unevaluated is better.